docs/CloudPermissions.md

```diff
@@ -0,0 +1,58 @@
+# Assign Required Cloud Permissions For Deploying Scalar DL
+
+This guide explains how to assign required permissions for the users for deploying Scalar DL on Azure.
+
+From the security perspective, it is better to give users limited privileges to protect the environment from unwanted activities.
```

Comment:
privileges
make it consistent.

Reply:
Sorry, Changed it to permissions, changed all privileges to permissions in 6239a04.